docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+
+## Error handling
+
+### Aborting with unhandled error
+
+The following code listing demonstrates how to abort the script with an unhandled error, which allows debugging with the stack trace of the crash during development.
+
+```go file=./cookbook/snippets/error-handling/unhandled/main.go
+```
+
+### Aborting gracefully
```

Comment:
```suggestion
### Abort pipeline gracefully
```

Reply:
Same, the meaning here is for "exiting" the "program".

Reply:
Isn't the "program" here specifically a Dagger pipeline?

Reply:
I meant "program" as the general concept here not a word suggestion. `go run` runs as a script, `go build` compiles in a program.

I don't see the script as the pipeline, I see the script as containing a pipeline. In this case the script only has a pipeline but I still don't equate the two. Our CI code using mage is more a script with multiple pipelines (one for each entrypoint).

Reply:
OK I see. Is this recipe explaining how to abort a pipeline (which is within a script) gracefully and then continue to run the remainder of the script, or abort the entire script (including one or more pipelines contained within it) gracefully?

Reply:
abort = exit with error

The use case here is to not let a pipeline error crash the program and instead handle it and terminate cleanly. 

A crash is abrupt, may not clean up system resources and displays an ugly stack trace to the user. Aborting with a "panic" is quick and easy, but this recipe shows how to do it cleanly.

Reply:
I changed it to "Terminate" and "program or script".

Reply:
I'm going to resolve this line since it's outdated, add new comment for follow-up feedback.